Web/Components/Pages/Books/AddBookScan.razor.css

```
@@ -0,0 +1,131 @@
+.add-book-scan-page {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: #000;
+}
+
+::deep .scanner-video-container {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)